app/components/primer/label_component.rb

```diff
@@ -37,23 +38,18 @@ class LabelComponent < Primer::Component
     #   <%= render(Primer::LabelComponent.new(title: "Label: Label")) { "Default" } %>
     #   <%= render(Primer::LabelComponent.new(title: "Label: Label", variant: :large)) { "Large" } %>
     #
-    # @example Deprecated schemes
-    #   <%= render(Primer::LabelComponent.new(title: "Label: Label", scheme: :orange)) { "Orange" } %>
-    #   <%= render(Primer::LabelComponent.new(title: "Label: Label", scheme: :purple)) { "Purple" } %>
-    #
     # @param title [String] `title` attribute for the component element.
     # @param scheme [Symbol] <%= one_of(Primer::LabelComponent::SCHEME_MAPPINGS.keys) %>
     # @param variant [Symbol] <%= one_of(Primer::LabelComponent::VARIANT_OPTIONS) %>
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(title:, scheme: nil, variant: nil, **system_arguments)
       @system_arguments = system_arguments
-      @system_arguments[:bg] = :blue if scheme.nil?
```

Comment:
this class has no effect in `Label`

Reply:
cc @simurai any idea how this ended up here?

Reply:
This was needed pre-PrimerCSS v15

Reply:
👍  Yeah, I think `.Label--blue` [changed](https://primer-css-git-mkt-color-modes-docs.primer.vercel.app/css/support/v16-migration#labels) `.Label--info`.